code/game/objects/items/crayons.dm

```diff
@@ -46,11 +46,11 @@
 	if(colour != "#FFFFFF" && shadeColour != "#000000")
 		colour = "#FFFFFF"
 		shadeColour = "#000000"
-		user << "You will now draw in white and black with this crayon."
+		to_chat(usr,"You will now draw in white and black with this crayon.")

```

Comment:
I believe that the previous code's choice in using `user` was the correct one.  I think changing it to `usr`, although unlikely to cause any obvious problems right away, isn't good.  We should continue to use `user` so that this proc will work correctly when invoked on behalf of someone.  This ought to be fixed when convenient.